### ISSUE TITLE
Use diff patches broadcast each commit

### ISSUE THREAD

**Issue by inverted-capital** [dreamcatcher-tech/artifact#109]
If each commit broadcasts a unified diff patch, then all watchers for all files can just watch for commit patches.

Watching for a specific file has the problem that we have to somehow signal to the system that we are interested in that specific file.

Or, we could broadcast to unique channels for each changed file, 

When FS writes the commit object, for no effort, it knows what files changed.
It should then either make a patch and broadcast it, or broadcast out on each channel, splices representing each changed file.

Basically, watching files presents a problem, where RTT to the db makes checking a file very slow.  Broadcasts need to be efficient to avoid chatter, and should be patches.
